src/java/org/apache/cassandra/repair/RepairMessageVerbHandler.java

```diff
@@ -184,11 +184,11 @@ public void doVerb(final Message<RepairMessage> message)
                         TableRepairManager repairManager = cfs.getRepairManager();
                         if (prs.isGlobal)
                         {
-                            repairManager.snapshot(desc.parentSessionId.toString(), prs.getRanges(), false);
```

Comment:
I am not completely sure this makes sense, because deterministicId() takes `RepairJobDesc#ranges` to create the name, but is it the same ranges as in `prs.getRanges()`?